.travis.yml

```diff
@@ -2,13 +2,7 @@ matrix:
   include:
     - os: osx
       language: objective-c
-      osx_image: xcode7.3
-      script:
-        - swiftlint
-        - set -o pipefail && xcodebuild test -project Phakchi.xcodeproj -scheme Phakchi -destination 'platform=iOS Simulator,name=iPhone 6' | xcpretty -c
-    - os: osx
-      language: objective-c
       osx_image: xcode8
```

Comment:
Unfortunately, Travis CI doesn't support Xcode 8.1 yet 😢 
I hope them to support next week.